src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts

```diff
@@ -57,6 +63,30 @@ class indexErrors extends viewModelBase {
             const remote = this.remoteLatestIndexErrorTime();
 
             return local !== remote;
+        });        
+      
+        this.allIndexesSelected = ko.pureComputed(() => { 
+            return this.allErroredIndexNames().length === this.selectedIndexNames().length;
+        });
+
+        this.clearErrorsBtnText = ko.pureComputed(() => {
+            if (this.allIndexesSelected() && this.allErroredIndexNames().length) {
+                return "Clear errors - All indexes";
+            } else if (this.selectedIndexNames().length){
+                return "Clear errors - Selected indexes";
+            } else {
+                return "Clear errors";
+            }
+        });
+
+        this.clearErrorsBtnTooltip = ko.pureComputed(() => {
+            if (this.allIndexesSelected() && this.allErroredIndexNames().length) {
+                return "Clear errors for all indexes";
+            } else if (this.selectedIndexNames().length){
+                return "Clear errors for selected indexes";
+            } else {
+                return "Select indexes to clear";
```

Comment:
tooltip is not visible when view is empty + text isn't true - no need to select anything - because then we delete all errors.